### ISSUE TITLE
discovering set to true if stopDevicesDiscovery() is called.

### ISSUE THREAD

**Issue by patricks** [kontaktio/kontakt-ios-sdk#27]
I have instantiated KTKDevicesManager in my ViewController.

`private var deviceManager: KTKDevicesManager!`

In the function `viewWillAppear()` I start the device discovery

``` swift
if deviceManager.discovering {
    return
}

deviceManager.startDevicesDiscoveryWithInterval(2)
```

and in the `viewWillDisappear()` function it stop it again.

This works without problems. But if I enter the ViewController a second time, the deivceManager.discovering Bool is true.

This should be false.

Is this a bug in the SDK version 1.2.2? Or did I forget something?

I have also created a demo project to reproduce the problem:

[github.com/patricks/kontaktio_discovery_problem/](https://github.com/patricks/kontaktio_discovery_problem/)

**Comment by stoprocent** [comment]
Hi @patricks, 
Good catch it's in fact a small bug. It will be fixed in next release, probably this week.

**Comment by stoprocent** [comment]
Closing now. It was fixed in 1.2.3 some time ago :)
